packages/fast-components/src/card/card.styles.ts

```diff
@@ -0,0 +1,14 @@
+import { css } from "@microsoft/fast-element";
+
+export const CardStyles = css`
+    :host {
+        --elevation: 4;
```

Comment:
Should we prefix our CSS variable names? @chrisdholt @nicholasrice has there been any thought about this regarding unification with other MS teams around variable names?

Reply:
We'll want to include the default style for `hidden` for the host. See the docs I wrote on styling in shadow dom. Also, consider adding `contain`.

Reply:
@nicholasrice has been working on the design tokens side of things, so he likely has more insight here. `elevation` and other variables I've used for now are just stubs in my mind. I do think we need to land a story here, but wanted to get things rolling while those convos are happening.

@EisenbergEffect can you elaborate on `contain` or point me to the relevant docs?

Reply:
https://github.com/microsoft/fast-dna/blob/master/packages/fast-element/docs/building-components.md#shadow-dom-styling

Reply:
Updated - very cool.

Reply:
I'll sync with Levi on whether they have landed a token prefix or not. I've noticed some "-ms-" prefixes in some of the prototyping they're doing, but I don't know if that is something they have actually landed.

My gut reaction is to *also* prefix, but if the intent is to arrive at a common token naming and providing theme interop that may not actually be desired.

Reply:
They have not landed that implementation detail so I think this is fine for now. I can write up a discussion issue to propose a decision tree for this, but we'll likely want to differentiate by whether the property used is a global design system token, or a more proprietary concept.